x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_schema.scss

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+.engineSchemaInnerTable .euiTable {
```

Comment:
Another option is using Emotion with euiTable `css` prop and overriding the background color.

Reply:
oh good deal. i like this better. thanks https://github.com/elastic/kibana/pull/153602/commits/4105a9217eaf3ca5d359a343fd3beee71a3d016f